assets/src/hooks/useDetours.ts

```diff
@@ -94,15 +94,15 @@ export const useActiveDetours = (
   enable: boolean = true
 ) => {
   const topic = "detours:active"
-  const [activeDetours, setActiveDetours] = useState<DetoursMap>({})
+  const [activeDetours, setActiveDetours] = useState<DetoursMap | null>(null)
```

Comment:
suggestion/question(non-blocking): what do you think about using `undefined` instead of `null`? 

To me, `undefined` means it hasn't been initializd yet, which correlates with a "loading" state, where `null` _to me_ would mean that the channel connected and got `null` as a response. 

I'm also _curious_ about something like this but not sure about it...
```suggestion
  const [activeDetours, setActiveDetours] = useState<DetoursMap | "loading">("loading")
```
I wish we had better types for "loading" states :/

Reply:
Hmmm I think I'm going to opt for `undefined` (good catch, thanks). Using "loading" led TS to struggle with the `{ ...activeDetours` logic because it couldn't ascertain whether it was working DetoursMap or a string (even if I incorporated conditionals to assert that `activeDetours != "loading"`)